src/bundle/ui-dev/src/modules/universal-discovery/bookmarks.tab.module.js

```diff
@@ -22,7 +22,11 @@ import { getIconPath } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scri
 import { getTranslator } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
 
 const BookmarksTabModule = () => {
-    const shouldRestorePreviousStateRef = useRef(true);
+    const didBookmarksMarkedLocationRef = useRef(false);
```

Comment:
I feel like there's something wrong with this name, so to be clear - you want to track if there is marked location that was set by bookmarks module?
In that case probably you should use present perfect and make it like `haveBookmarksMarkedLocationRef` :P 
My main problem here is that in other places "MarkedLocations" is adjective with noun, but here I think you used it as verb with noun, which is misleading, I think.

Reply:
I changed this name to `isMarkedLocationSetByBookmarksRef`. :)